src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/DescriptionAddendum.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.dto
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+@Schema(description = "Addendum to the description", accessMode = Schema.AccessMode.READ_ONLY)
+@JsonInclude(JsonInclude.Include.ALWAYS)
+data class DescriptionAddendum(
+  @Schema(description = "Username of user who added this addendum", example = "USER_1")
+  val createdBy: String,
```

Comment:
@xoen we will also want to have the first and last name of the staff member, as the appended data stores their name and NOT their username - to populate this field we are going to have to do a reverse lookup and potentially we either may not find the record or there will be more than one result.  My feeling is that for migrated records we just store the first  and last name and leave the createdBy blank but for new records we can populate username and their name (like we do for the staff involvement )